jobs/config.json

```diff
@@ -9229,30 +9240,53 @@
   },
   "ci-kubernetes-node-kubelet-benchmark": {
     "args": [
-      "--properties=test/e2e_node/jenkins/benchmark/jenkins-benchmark.properties"
+      "--deployment=node",
+      "--gcp-project=k8s-jkns-ci-node-e2e",
+      "--gcp-zone=us-central1-f",
+      "--node-args=--image-config-file=../test-infra/jobs/benchmark-config.yaml",
+      "--node-test-args=--feature-gates=DynamicKubeletConfig=true --kubelet-flags=\"--cgroups-per-qos=true --cgroup-root=/\"",
+      "--node-tests=true",
+      "--provider=gce",
+      "--test_args=--nodes=1 --skip=\"\\[Flaky\\]\"",
+      "--timeout=65m"
     ],
-    "scenario": "kubernetes_kubelet",
+    "scenario": "kubernetes_e2e",
     "sigOwners": [
-      "UNKNOWN"
+      "sig-node"
     ]
   },
   "ci-kubernetes-node-kubelet-conformance": {
     "args": [
-      "--properties=test/e2e_node/jenkins/conformance/jenkins-conformance.properties",
-      "--script=test/e2e_node/jenkins/conformance/conformance-jenkins.sh"
+      "--deployment=node",
+      "--gcp-project=k8s-jkns-ci-node-e2e",
+      "--gcp-zone=us-central1-f",
+      "--node-args=--image-config-file=../test-infra/jobs/benchmark-config.yaml",
```

Comment:
couple years later I'm here wondering why the benchmark-config is used for the conformance suite?